src/materialized/ci/Dockerfile

```diff
@@ -13,4 +13,8 @@ RUN apt-get update && apt-get -qy install ca-certificates
 
 COPY kdestroy kinit materialized /usr/local/bin/
 
+RUN mkdir -p mzdata && chown 9000:9000 mzdata
+
+USER 9000:9000
```

Comment:
Randomly selecting a UID like this causes weird problems (no /etc/passwd entry). Best practice AFAIK is to use `useradd` to actually add a `materialized` user. 

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user

Reply:
To be specific, you probably want something like the following:

```
RUN groupadd --gid 9000 mtrlz
RUN useradd --uid 9000 --gid 9000 -d /home/mtrlz mtrlz
RUN chown -R mtrlz:mtrlz /home/mtrlz

RUN mkdir -p mzadata && chown mtrlz:mtrlz mzdata
```

This will create a user and group, with an `/etc/passwd` entry and a home directory in case it needs a writeable directory within the container.